Hack to allow build tracker libs correctly with older libs installed in system
--- tracker-1.12.0/src/libtracker-miner/Makefile.am.orig	2016-11-27 20:33:39.000000000 +0100
+++ tracker-1.12.0/src/libtracker-miner/Makefile.am	2017-05-30 22:14:40.255185439 +0200
@@ -111,8 +111,8 @@
 
 libtracker_miner_@TRACKER_API_VERSION@_la_LIBADD =     \
 	libtracker-miner-private.la \
-	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	$(top_builddir)/src/libtracker-sparql-backend/libtracker-sparql-@TRACKER_API_VERSION@.la \
+	$(top_builddir)/src/libtracker-common/libtracker-common.la \
 	-lm                                            \
 	$(BUILD_LIBS)                                  \
 	$(LIBTRACKER_MINER_LIBS)
